website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -0,0 +1,185 @@
+    - Required buckets are PREFIX-tf-state, PREFIX-glacier, PREFIX-internal, PREFIX-private, PREFIX-protected, PREFIX-public, and PREFIX-orca
+    :::tip
+    An example command for creating a bucket in us-west-2. Remember to run ```aws configure``` first.
+    ```commandline
+    aws s3api create-bucket --bucket PREFIX-tf-state --profile default --region us-west-2 --create-bucket-configuration "LocationConstraint=us-west-2"
+    ```
+
+## [Cumulus](https://nasa.github.io/cumulus/docs/deployment/deployment-readme)
+- If creating a realistic setup with multiple OUs, apply [Create the ORCA Archive Bucket](creating-orca-glacier-bucket.md) to your PREFIX-glacier.
+- Run
+  ```commandline
+  aws dynamodb create-table --table-name PREFIX-tf-locks --attribute-definitions AttributeName=LockID,AttributeType=S --key-schema AttributeName=LockID,KeyType=HASH --billing-mode PAY_PER_REQUEST --region us-west-2
+  ```
+- Run
+  ```commandline
+  aws s3api put-bucket-versioning --bucket PREFIX-tf-state --versioning-configuration Status=Enabled
+  ```
+  :::warning
+  We presently have no instructions for creating a VPC, Subnet, Security Group, or PostgreSQL database.
+  Borrow them from an existing setup.
+  :::
+- Go to [this repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-template-deploy/browse) and download.
+- Unzip.
+- Remove the '.example' on terraform.tf and terraform.tfvars files in data-persistence-tf, cumulus-tf, and rds-cluster-tf.
+- In each terraform.tf and terraform.tfvars, use your own prefix, region, vpc id, and subnet ids.
+  :::warning
+  The region and prefix are not always in simple variables. Do a global search for 'PREFIX' and 'us-east-1'.
+  :::
+  :::warning
+  Only use the non-lambda subnet id in the data-persistence-tf/terraform.tfvars. In cumulus-tf use both.
+  :::
+  :::warning
+  Overwrite the ```orca-sandbox``` in ```orca-sandbox-tf-locks``` with your prefix as well.
+  :::
+- In rds-cluster-tf/terraform.tfvars
+    - Use values of your choice for ```db_admin_username``` and ```db_admin_password```
+    - Set ```tags``` to ```{ "Deployment" = "PREFIX" }```
+    - Set ```permissions_boundary_arn``` to ```arn:aws:iam::YOUR ACCOUNT ID:policy/NGAPShRoleBoundary```
+    - Add ```rds_user_password = "CumulusD3faultPassw0rd"``` and change as desired.
+    - Set ```provision_user_database``` to ```true```
+    - Set ```cluster_identifier``` to ```"PREFIX-cumulus-db"```
+
+- In rds-cluster-tf/terraform.tf
+    - Set ```bucket``` to ```"PREFIX-tf-state"```
+    - Set ```key``` to ```"PREFIX/cumulus/terraform.tfstate"```
+    - Set ```dynamodb_table``` to ```"PREFIX-tf-locks"```
+
+- Go to https://github.com/asfadmin/CIRRUS-core/blob/master/Dockerfile and download the file to the same folder as your downloaded repo and orca folder.
+  :::tip
+  Make sure that no extension is added.
+  :::
+- Open a commandline in the same folder.
```

Comment:
Just a thought, would it make sense to do docker volumes here instead of mounting to the windows file system? Maybe get around some of the Windows return line and other issues?
```
docker volume create cumulus-orca-data
docker volume create aws_config

docker run -it --rm -v  cumulus-orca-data:/CIRRUS-core -v aws_config:/root/.aws orca /bin/bash
```

The downs side to this approach is you would need to download those file in those containers (clone, etc) instead of the Windows local files system. I believe we can still interact with everything through VS Code by connecting to the container with VS Code.

Reply:
VS Code is incredibly buggy. The version on my machine now crashes every time I attempt to connect to a container.

Reply:
Well that is not cool. It was just a thought to not even have to dip into the windows file system at all.